include/dxc/dxcapi.h

```diff
@@ -13,8 +13,18 @@
 #ifndef __DXC_API__
 #define __DXC_API__
 
-#ifndef DXC_API_IMPORT
-#define DXC_API_IMPORT __declspec(dllimport)
+#ifdef _WIN32
```

Comment:
Don't you need compiler-specific switches here? Do all windows compilers support `__declspec(dll*)` and all non-windows compiler support `__attribute__((visibility))`?

Reply:
MinGW and clang on Windows also supports __declspec(dll*). So it become a platform thing.